docs/spec/app.json

```diff
@@ -48,7 +48,7 @@
                     "maxLength": 1024
                 }
             },
-            "required": ["name", "version"]
```

Comment:
version type shouldn't be also ["string", "null"] ?

Reply:
will add it